YIF.Core.Domain/ApiModels/Validators/PaymentAndEducationFormsPostApiModelValidator.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FluentValidation;
+using System.Linq;
+using YIF.Core.Data;
+using YIF.Core.Data.Entities;
+using YIF.Core.Domain.ApiModels.ResponseApiModels.EntityForResponse;
+
+namespace YIF.Core.Domain.ApiModels.Validators
+{
+    public class PaymentAndEducationFormsPostApiModelValidator : AbstractValidator<PaymentAndEducationFormsPostApiModel>
+    {
+        private readonly EFDbContext _context;
+        private readonly string NotValid = "Such {PropertyName} isn't valid";
```

Comment:
I think you shoud use resouceManager instead of hardcode string there

Reply:
done